jscomp/bsb/bsb_build_util.ml

```diff
@@ -124,6 +124,7 @@ let resolve_bsb_magic_file ~cwd ~desc p : result =
         else rest
       in
       (* let p = if Ext_sys.is_windows_or_cygwin then Ext_string.replace_slash_backward p else p in *)
+      (* TODO: it's unclear whether we should be calling Bsb_pkg.resolve_package here instead. Let's discuss in PR #304 *)
```

Comment:
I missed this one the first time around, and now I don't remember whether it was intentional or not 🤦

Reply:
I believe this is at least used to resolve ppx paths, e.g. `some-ppx-in-node-modules/ppx.exe`. So I believe it should be here too.